docs/dev/releasing.md

```diff
@@ -21,33 +21,35 @@ This document guides maintainers through the process of creating an Envoy Gatewa
    ```
 
 5. Push the branch to the Envoy Gateway repo.
-6. Create a topic branch and update the release tag references in the [Quickstart Guide][]. See [PR 481][] as
-   a reference for the required changes.
+6. Create a topic branch and update the release tag references in the [Quickstart Guide][].
+
+   ```shell
+   make update-quickstart TAG=v0.3.0
+   ```
+
 7. Sign, commit, and push your changes to your fork. Send a PR to get your changes merged into the release branch.
    Do not proceed until your PR is merged.
-8. Confirm that the [release workflow][] for your PR completed successfully.
-9. Tag the head of your release branch with the release tag. For example:
+8. Tag the head of your release branch with the release tag. For example:
 
    ```shell
    git tag -a v0.3.0 -m 'Envoy Gateway v0.3.0 Release'
    ```
 
    __Note:__ The tag version differs from the release branch by including the `.0` patch version.
 
-10. Push the tag to the Envoy Gateway repository.
+9. Push the tag to the Envoy Gateway repository.
 
     ```shell
-    git push --tags
+    git push v0.3.0
```

Comment:
Do not use `--tags`, it will cause something wrong if we have create some other git tags locally.